### ISSUE TITLE
Review dilated MP terms

### ISSUE THREAD

**Issue by anna-anagnostop** [mgijax/mammalian-phenotype-ontology#3590]
We need to go through all MP terms that use 'dilated' in the primary term label and make sure that:

- primary term label uses 'dilated' for consistency (see [ascending aorta dilation](http://www.informatics.jax.org/vocab/mp_ontology/MP:0010470), [pancreatic acinus dilation](http://www.informatics.jax.org/vocab/mp_ontology/MP:0014232))
- 'dilation' is added as an exact synonym
- primary term label uses singular form (e.g. change [dilated brain ventricles](http://www.informatics.jax.org/vocab/mp_ontology/MP:0012676) to dilated brain ventricle)
- plural form is added as an exact synonym 
- amend definition to read as "one or more" for multiple structures (e.g. [dilated hair follicles](http://www.informatics.jax.org/vocab/mp_ontology/MP:0010699)) or "one or both" for paired structures (e.g. [dilated heart atrium](http://www.informatics.jax.org/vocab/mp_ontology/MP:0003140), [dilated heart ventricle](http://www.informatics.jax.org/vocab/mp_ontology/MP:0008022))

**Comment by anna-anagnostop** [comment]
Note to self: stopped at [dilated pulmonary alveolar ducts](http://www.informatics.jax.org/vocab/mp_ontology/MP:0010908)
